### PR TITLE
flac2wav: Add converter example using the FLAC decoder and WAV encoder.

### PR DESCRIPTION
**Pull request by mewmew**

### CODE REVIEW

File: azul3d_flac2wav/azul3d_flac2wav.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	// Copy samples from the FLAC decoder to the WAV encoder.
+	if _, err = audio.Copy(enc, dec); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// helper functions.
+//
```
Comment (slimsag) [inline azul3d_flac2wav/azul3d_flac2wav.go]: I think in general this sort of comment is not useful.

**Comment by slimsag** [comment]
I left a minor comment, if you would like to commit to master an update please do so. Apologies for it taking so long for me to get to this! LGTM

**Comment by mewmew** [comment]
Thanks for merging the pull request! Don't worry, I've also been busy lately. Still it's good to see you back playing with Azul3d :)

I tried committing the change you suggested, but was unable due to permissions on the `examples` repository:

```
ERROR: Permission to azul3d/examples.git denied to mewmew.
```

**Comment by slimsag** [comment]
Good to be back :smile: 

I've updated perms to give all contributors access to examples repo, too. could you try again?

**Comment by mewmew** [comment]
Thanks! It works now :) Welcome back!

**Pull request merged**